app/views/reservations/_form.html.erb

```diff
@@ -41,9 +41,6 @@
     </div>
```

Comment:
We should also remove the current notes from the reservations form partial since we're no longer adding to them from that form.